From 9f333d1422820c966ccd0b834f18009058aaad75 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sat, 09 Jul 2022 17:40:28 +0300
Subject: [PATCH] [SW-94367] gaudi3: perform iatu and boot termination cfg for slave die

As recommended by arch team, this patch performs IATU configuration
and clearing of boot termination for slave die as part of PCIe
routine.

Change-Id: I521dd368593bc298643d3a494dae3b5e823f10cc
---

diff --git a/common/pcie.c b/common/pcie.c
index 95e8a83..74b10be 100644
--- a/common/pcie.c
+++ b/common/pcie.c
@@ -135,11 +135,9 @@
 		}
 
 		/* Configure IATU */
-		if (!arcboot_rom_global.die_id) {
-			arcboot_rom_status_update(PROGRESS_ARCBOOT_ROM_PCIE_LINK_IATU_CONFIG);
-			pcie_config_iatu();
-			pcie_config_boot_termination();
-		}
+		arcboot_rom_status_update(PROGRESS_ARCBOOT_ROM_PCIE_LINK_IATU_CONFIG);
+		pcie_config_iatu();
+		pcie_config_boot_termination();
 
 		/* enable flr handling by HW state machine */
 		enable_flr_handling_fsm();
